tactics/class_tactics.ml

```diff
@@ -1341,7 +1341,7 @@ module Search = struct
                                             | Some i -> str ", with depth limit " ++ int i));
     tac
 
-  let run_on_evars p evm tac =
+  let run_on_evars p evm tac env =
```

Comment:
I assume sigma and evm are the same so that you want the following prototype :
`let run_on_evars env evm p tac =`